community/governance/index.md

```diff
@@ -5,4 +5,125 @@ sidebar:
   nav: community
 ---
 
-* [SIGs](/community/governance/sigs)
+{% include toc %}
+
+The Spinnaker OSS project is run by a group of appointed members who hold specific roles, managed by the technical oversight committee. The steering committee manages the governance structure of the project, revisiting and refining it as needs evolve.
```

Comment:
"is run by a group of appointed members"
Is this the steering committee? The way this first sentence sounds, the (steering committee?) is managed by the technical oversight committee.